LWMS ID 11022: Fixed worpress db user error in Lab 4

plforacle opened the pull request:
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

anooshapilli reviewed (commented):
@plforacle - There are new files created and updated in the folders other than WMS ID 11022 workshop folder. Could you share more details about these changes and the ID of those workshops?

plforacle commented:
Hi Anoosha,

I am trying to fix a problem for the WordPress workshop. A user wrote to me and complained about it. Which new files are you talking about? I only want to deploy the LWMS ID 11022: in the folder plforacle/database/tree/main/mds-wordpress.

Thank you,

Perside Lafrance Foster | MySQL Principal Partners Solution Engineer
email: ***@***.******@***.***>
phone: 1.786.877.9063
Oracle  Florida

From: Anoosha Pilli ***@***.***>
Sent: Friday, May 12, 2023 5:43 PM
To: oracle-livelabs/database ***@***.***>
Cc: Perside Foster ***@***.***>; Mention ***@***.***>
Subject: [External] : Re: [oracle-livelabs/database] LWMS ID 11022: Fixed worpress db user error in Lab 4 (PR #257)


@anooshapilli commented on this pull request.

@plforacle<https://urldefense.com/v3/__https:/github.com/plforacle__;!!ACWV5N9M2RV99hQ!JDggYimQUkJUdK3IpMudutu25vEGMBT2lkIUfaTnDaThMztrnNkIWtVc8bBHIav98DUfUBMTXrrqFA5ljYmONBMhigtx$> - There are new files created and updated in the folders other than WMS ID 11022 workshop folder. Could you share more details about these changes and the ID of those workshops?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/oracle-livelabs/database/pull/257*pullrequestreview-1425187814__;Iw!!ACWV5N9M2RV99hQ!JDggYimQUkJUdK3IpMudutu25vEGMBT2lkIUfaTnDaThMztrnNkIWtVc8bBHIav98DUfUBMTXrrqFA5ljYmONHkFyRGT$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AMYQBSTTQFTWGAM3MMCLTILXF2VGVANCNFSM6AAAAAAX77YQUE__;!!ACWV5N9M2RV99hQ!JDggYimQUkJUdK3IpMudutu25vEGMBT2lkIUfaTnDaThMztrnNkIWtVc8bBHIav98DUfUBMTXrrqFA5ljYmONBHOCpZP$>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

plforacle commented:
Several of the changes are for Oracle cloud world ( I am updating 4 different workshops for that event) .... I don't want to publish them yet. They are not part of the WordPress folder (LIveLab Workshop). I will create a new PR for the WordPress LiveLab.
